### DISCUSSION TITLE
How do I change the background and axis colors?

### DISCUSSION THREAD

**Discussion by jdnec** [alandefreitas/matplotplusplus#229] [Q&A]
I can't find anywhere how to change the background and the axis colors. 
I want a chart that looks similars to the ones in the guides: 
https://alandefreitas.github.io/matplotplusplus/plot-types/line-plots/line-plot/
Basically black background with white axis. 

Edit:
According to another post I need to make changes in some header files (axes_object.h, axes_type...) but I´m very new to C++ and I really don´t know what I should modify or how. If someone could tell me the solution I would be greatly thanksful for that. 
Basically the only two things I want to change is the background color from white to black, and the axis colors and text colors from black to white. Just like this image:
![barchart](https://user-images.githubusercontent.com/99940200/154703051-387e1884-90f5-4108-8358-ddbceb7c8ec7.svg)
